docs/troubleshooting_pi-server_integration.rst

```diff
@@ -0,0 +1,180 @@
+If PI Web API is configured for *Basic* authentication a prompt asking user name/password like the following one will appear:
+
+|img_002|
+
+**NOTE:**
+
+- *The same user name/password configured in Fledge in the north instance should be used.*
+
+The *PI Web API* *OMF* plugin must be installed to allow the integration with Fledge, in this screen shot the 4th row shows the
+proper installation of the plugin:
+
+|img_001|
+
+Select the item *System* to verify the installed version:
+
+|img_010|
+
+Commands to check the PI WEB API
+--------------------------------
+
+Open the PI Web API URL and drill drown into Data Archive and into Asset Framework to verify the proper configuration on the PI-Server side, also in terms of granted permissions.
+
+**Data Archive drill down**
+
+following the path *DataServers* -> *Points*:
+
+|img_004|
+
+|img_005|
+
+you should be able to browse the *PI Points* page and see your *PI Points* if some data was already sent:
+
+|img_006|
+
+**Asset Framework drill down**
+
+following the path *AssetServers* -> Select the *Instance* -> Select the proper *Databases* -> drill down into the AF hierarchy up to the required level -> *Elements*:
+
+|img_007|
+
+*selecting the instance*
+
+|img_008|
+
+*selecting the database*
+
+|img_009|
+
+proceed with the drill down operation up to the desired level/asset.
+
+Error messages an the related causes
+------------------------------------
+
+Some error messages and the related causes:
+
+.. list-table::
+    :widths: 50 50
+    :header-rows: 1
+
+    * - Message
+      - Cause
+    * - North_Readings_to_PI[20884]: WARNING: Error in retrieving the PIWebAPI version, The **PI Web API server is not reachable**, verify the network reachability
+      - Fledge is not able to reach the machine in which PI-Server is running due to a network problem of a firewall restriction.
+    * - North_Readings_to_PI[5838]: WARNING: Error in retrieving the PIWebAPI version, **503 Service Unavailable**
+      - Fledge is capable to reach the machine in which PI-Server is executed but the PI Web API is not running.
+    * - North_Readings_to_PI[24485]: ERROR: Sending JSON data error : **Container not found**. 4273005507977094880_1measurement_sin_4816_asset_1 - WIN-4M7ODKB0RH2:443 /piwebapi/omf
+      - Fledge is able to interact with PI Web API but there is an attempt to store data in a PI Point that does not exist.
+
+Possible solutions to common situations
+---------------------------------------
+
+- **Recreate a single or a sets of PI-Server objets and send again all the data for them to the PI Server on the Asset Framework hierarchy level**
+    procedure:
+        - disable the 1st north instance
+        - delete properly the objects on the PI Server, AF + Data archive, that we intend to recreate or were eventually partially deleted
+        - create a new **DISABLED** north instance using a new/unique name and having the same AF hierarchy of the 1st north instance
```

Comment:
create a new **DISABLED** north instance using a new, unique name and having the same AF hierarchy as the 1st north instance